testing/go/dolt_functions_test.go

```diff
@@ -20,7 +20,2381 @@ import (
 	"github.com/dolthub/go-mysql-server/sql"
 )
 
-func TestDoltFunctions(t *testing.T) {
+func TestDoltAdd(t *testing.T) {
```

Comment:
(minor) This test file is getting pretty large – over 3k lines now. Seems like it should be multiple files.

Reply:
Thought about splitting it up, but it would end up creating a lot of test files in the directory, making it a little harder to find any one test. Would probably be better to have a subdirectory in the `test/go` folder, but I think we'd have to modify the framework a little bit to do so. We have the same problem with `pgcatalog_test.go` as well, that one is almost 5k lines.